documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc

```diff
@@ -141,8 +151,8 @@ class MainLayout extends PolymerElement {
 customElements.define(MainLayout.is, MainLayout);
 ----
 
-To use the template file, you also need a basic template component class with a `@JsModule` annotation for the template file.
-Right now you need to also implement the `RouterLayout` interface:
+
+*Example*: Basic `MainLayout` server class that imports the Polymer template and implements `RouterLayout`.
```

Comment:
The original remark importance is a bit lost by moving it into an 'Example' block. Many quick-readers skip anything that's after the 'Example' block. I'd keep the original, perhaps with a bit of rewording.

Reply:
Yes I agree. Will introduce back the remark.